tests/values.go

```diff
@@ -498,3 +514,89 @@ func ReadValuesAsMap(path string) (map[string]interface{}, error) {
 
 	return values, nil
 }
+
+// ReadValuesAsMap reads the values.yaml from a string.
+func ReadValuesAsMap(valuesStr string) (map[string]interface{}, error) {
+	var values map[string]interface{}
+	decoder := yaml.NewYAMLToJSONDecoder(strings.NewReader(valuesStr))
+	err := decoder.Decode(&values)
+	if err != nil {
+		return nil, fmt.Errorf("error decoding yaml: %w", err)
+	}
+
+	return values, nil
+}
+
+func ReadChartValues(t testing.TB, chart *chart.Chart, valuesStr string) chartutil.Values {
```

Comment:
is ReadChartValues and ReadValuesAsMap actually used anywhere? I can't see where they're being used

Reply:
I used these in a prior iteration when I was embedding a yaml string instead of using the `CoderValues` type. I'll remove these